CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 #### Upcoming Changes
 
+* Implement test for LinearSplit hint [#1118](https://github.com/lambdaclass/cairo-rs/pull/1118)
```

Comment:
I think it is not necesery to update the changelog
Since we are just adding tests